presto-product-tests/bin/product-tests-suite-3.sh

```diff
@@ -19,5 +19,10 @@ presto-product-tests/bin/run_on_docker.sh \
     -g storage_formats,cli,hdfs_impersonation,authorization \
     || exit_code=1
 
+presto-product-tests/bin/run_on_docker.sh \
```

Comment:
I guess you want to use `product-tests-suite-6-non-generic.sh`. `product-tests-suite-3` is executed for CDH and HDP, and I think you don't use hadoop at all, do you?